website/payments/api/v2/admin/serializers/payable_create.py

```diff
@@ -0,0 +1,12 @@
+from rest_framework.fields import ChoiceField
+from rest_framework.serializers import Serializer
+
+from payments.models import Payment
+
+
+class PayableCreateAdminSerializer(Serializer):
+    """Serializer to create a payment from a payable."""
+
+    payment_type = ChoiceField(
+        choices=[Payment.TPAY, Payment.CASH, Payment.CARD, Payment.WIRE]
```

Comment:
This does allow admins to create Thalia Pay payments for payables (for other users). Do we want to allow that? Right now, we don't...

Reply:
This is my last review comment, I promise

Reply:
Yes we do want this, because this is what is necessary for all the payables to be paid from the app as admin. It replaces the complete 'set payment to NONE/CARD/WIRE etc and then it magically does this work for you. Example of usage is #1684.

And that is also why I added the `can_manage` part to the Payable, it determines if someone can create payments for said Payable, to lock it down.

Reply:
I explicitly mean Thalia Pay payments. Right now, we (deliberately?) do not allow regular admins (like EVC members) to create TPay payments on behalf of others: only users themselves should be able to do that. I do see that Payment.TPAY is an actual payment type, but I don't think we should allow writing it.

Reply:
Ah I think I misunderstood what you meant. You're right, this option should not exist.